moveit_demo_nodes/run_move_group/launch/run_move_group.launch.py

```diff
@@ -5,111 +5,35 @@
 from launch.actions import ExecuteProcess
 from ament_index_python.packages import get_package_share_directory
 import xacro
-
-
-def load_file(package_name, file_path):
-    package_path = get_package_share_directory(package_name)
-    absolute_file_path = os.path.join(package_path, file_path)
-
-    try:
-        with open(absolute_file_path, "r") as file:
-            return file.read()
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
-        return None
-
-
-def load_yaml(package_name, file_path):
-    package_path = get_package_share_directory(package_name)
-    absolute_file_path = os.path.join(package_path, file_path)
-
-    try:
-        with open(absolute_file_path, "r") as file:
-            return yaml.safe_load(file)
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
-        return None
+from moveit_configs_utils.moveit_configs_builder import MoveItConfigsBuilder
 
 
 def generate_launch_description():
-
-    # planning_context
-    robot_description_config = xacro.process_file(
-        os.path.join(
-            get_package_share_directory("moveit_resources_panda_moveit_config"),
-            "config",
-            "panda.urdf.xacro",
-        )
+    moveit_config = (
+        MoveItConfigsBuilder("moveit_resources_panda")
+        .robot_description(file_name="panda.urdf.xacro")
```

Comment:
I think the `str` args here should be filepaths within the package instead of just the name. In `MoveItConfigsBuilder`, this filename expands to `config/panda.urdf.xacro`. That might be OK for all the moveit config packages, but if somebody wanted to use this for their own (e.g. modified) config package, they would have to put files in specific places or modify the configs builder